<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomShapeDialog</class>
 <widget class="QDialog" name="CustomShapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom Shape Selector</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="listArea" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>150</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearShape">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Shape</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editShape">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Shape</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="polygonEditArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomShapeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
